packages/next/server/config-shared.ts

```diff
@@ -544,7 +544,9 @@ export const defaultConfig: NextConfig = {
     images: {
       remotePatterns: [],
     },
+    swcTraceProfiling: false,
     forceSwcTransforms: false,
+    swcPlugins: [],
```

Comment:
```suggestion
    swcPlugins: undefined,
```
Should the default be undefined here to differentiate a user providing an empty array?

Reply:
I thought undefined is not supported, but it should be better if it's supported.